codility_training/lessons.lesson07.StacksandQueues.Fish/sangmandu.py

```diff
@@ -0,0 +1,26 @@
+# you can write to stdout for debugging purposes, e.g.
+# print("this is a debug message")
+
+def solution(A, B):
+    # write your code in Python 3.6
+    pass
+    S = []
+    cnt = 0
+
+    for i in range(len(A)):
+        if B[i] == 1:
+            S.append(A[i])
+        else:
+            while len(S) != 0:
+                fish = S.pop()
+                if fish > A[i]:
+                    S.append(fish)
+                    break
+            if (len(S) == 0):
+                cnt += 1

```

Comment:
```suggestion
    for i, v in enumerate(A):
        if B[i] == 1:
            S.append(v)
            continue;
        while len(S) != 0:
            fish = S.pop()
            if fish > v:
                S.append(fish)
                break
        if (len(S) == 0):
            cnt += 1
```
앞선 리뷰와 마찬가지로 if의 depth를 최대한 줄이면 좋습니다!

Reply:
그리고 v를 사용하는게 적은 호출을 할 수 있겠군요! 감사합니다